src/server/hset_family_test.cc

```diff
@@ -541,4 +541,30 @@ TEST_F(HSetFamilyTest, KeyRemovedWhenEmpty) {
   test_cmd([&] { EXPECT_THAT(Run({"HSTRLEN", "a", "afield"}), IntArg(0)); }, "HSTRLEN");
 }
 
+TEST_F(HSetFamilyTest, HRandFieldRespFormat) {
+  absl::flat_hash_map<std::string, std::string> expected{
+      {"a", "1"},
+      {"b", "2"},
+      {"c", "3"},
+  };
+  Run({"HELLO", "3"});
+  EXPECT_THAT(Run({"HSET", "key", "a", "1", "b", "2", "c", "3"}), IntArg(3));
+  auto resp = Run({"HRANDFIELD", "key", "3", "WITHVALUES"});
+  EXPECT_THAT(resp, ArrLen(3));
+  for (const auto& v : resp.GetVec()) {
+    EXPECT_THAT(v, ArrLen(2));
+    EXPECT_THAT(v.GetVec()[0], AnyOf("a", "b", "c"));
```

Comment:
why not `const auto &v = v.GetVec()`. You called it 3 times :smile:

Reply:
fixed